pkg/commands/run.go

```diff
@@ -413,6 +417,14 @@ func (e *Executor) createPrinter() (printers.Printer, error) {
 	return p, nil
 }
 
+func (e *Executor) postProcess(issues []result.Issue) {
```

Comment:
I was about to add the comment for Processor pattern to make it more composable, but then saw your #1226 

Which approach do you prefer?

Reply:
+1 for the #1226 
It looks more appropriate